lib/galaxy_test/driver/driver_util.py

```diff
@@ -517,8 +517,8 @@ def serve_webapp(webapp, port=None, host=None):
         try:
             server = httpserver.serve(webapp, host=host, port=port, start_loop=False)
             break
-        except socket.error as e:
-            if e[0] == 98:
+        except OSError as e:
```

Comment:
If you are interested in keeping Python 2.7 compatibility, this should work on 2.7 and 3.5-3.8:

```suggestion
        except socket.error as e:
```